lib/whitehall_importer/sync.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module WhitehallImporter
+  class Sync
+    attr_reader :whitehall_import
+
+    def self.call(whitehall_import)
+      new(whitehall_import).call
+    end
```

Comment:
If you go for:

```
    def self.call(*args)
      new(*args).call
    end
```

Then we won't have to worry about keeping the attributes of this method in sync with initialize.

Reply:
Done